examples/tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+out*
+expected
```

Comment:
This doesn't seem necessary, right @dchw?

Reply:
At least as of my last commit; the expected listing was exported as a separate artifact as just a bare file named `expected`.

Reply:
Right... but that wasn't actually exported on the host, right? It was exported in the environment of the Earthfile that runs the test.

Reply:
Looking at how you tweaked the tests to save each, we should change it to `expected*`.

The file in general is here if you run them locally (outside the main tests `Earthfile`) to prevent any accidents committing the test results.

Reply:
I can live with removing it and just being careful; if it bothers you.

Reply:
:+1: